docs/documentation/quartz-3.x/how-tos/multiple-triggers.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Multiple Triggers
+---
+
+One nice thing about Quartz is that you can split the job registration from the 
+triggers. One nice use case is being able to split out the JobData for each
+trigger. 
+
+Below, we have two triggers, each with their own set of data, but
+we only had to register the one job.
+
+
+```csharp
+public Task DoSomething(IScheduler schedule, CancellationToken ct)
+{
+    var job = JobBuilder.Create<AnExampleJob>()
+                        .WithIdentity("customer-process", "group")
+                        .Build();
+    
+    var replace = true;
+    var durable = true;
+    await schedule.AddJob(job, replace, durable, ct);
+
+    // Trigger 1
+    var jobData1 = new JobDataMap { { "CustomerId", 1 } };
+    await scheduler.TriggerJob(new JobKey("customer-process", "group"), jobData1, ct);
+
+    // Trigger 2
+    var jobData2 = new JobDataMap { { "CustomerId", 2 } };
```

Comment:
Quartz has configuration recommendation to have `useProperties` in configuration which validates that only string keys and values are being used for serialization safety (false by default as legacy). It's debatable if this should not raise an error if primary value types like int are being used. Maybe would need three modes (string-only, value-type-only, anything-and-prepare-for-serialization-issues).

Reply:
I'll change my code to use `string`. I'd like to keep the examples as much in the happy path as possible.